lib/compliance/triggers/consts.js

```diff
@@ -19,7 +19,9 @@ const REQUIREMENTS = {
   FACEPHOTO: 'facephoto',
   SUSPEND: 'suspend',
   BLOCK: 'block',
-  US_SSN: 'usSsn'
+  US_SSN: 'usSsn',
+  EXTERNAL: 'external',
+  CUSTOM: 'custom'
 }
 
 const ORDERED_REQUIREMENTS = [
```

Comment:
Do you know why custom requirements aren't part of this list?

Reply:
It was added because 'custom' is now a trigger tier, instead of extrapolating that data based on the validity of a UUID field. Comment above has more context to this change